packages/botonic-react/src/components/carousel.jsx

```diff
@@ -1,7 +1,26 @@
 import React from 'react'
+import { Message } from './message'
 import { isBrowser, isNode } from '@botonic/core'
 
 export const Carousel = props => {
+  let content = ''
+  if (isBrowser())
```

Comment:
```suggestion
  if (isBrowser()) {
```

Reply:
forget about this one if you havent' applied, not sure if "prettier" wants it like this :-)

Reply:
Yes, prettier accept it! I will add it now ;)